samwise.js

```diff
@@ -1,4 +1,4 @@
-const objectKeys = Object.keys(object);
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?